mmpose/datasets/datasets/body3d/body3d_base_dataset.py

```diff
@@ -21,6 +21,18 @@ class Body3DBaseDataset(Dataset, metaclass=ABCMeta):
         img_prefix (str): Path to a directory where images are held.
             Default: None.
         data_cfg (dict): config
+            - num_joints: Number of joints.
+            - seq_len: Number of frames in a sequence. Default: 1.
+            - seq_frame_interval: Extract frames from the video at certain
+                intervals. Default: 1.
+            - causal: Whether to use causal convolutions. If set to True, the
```

Comment:
It's better to avoid using the term "causal convolutions" here since it's not about the dataset but the model. I think it's okay to remove the first sentence.